infra/external-api/rate/models/exchange-rate-api.go

```diff
@@ -1,4 +1,4 @@
-package rate
+package models
```

Comment:
```suggestion
package model
```
Зазвичай в go називають пакети в однині, у випадку якщо це можливо.
https://rakyll.org/style-packages/#:~:text=IT%2C%20AVOID%20GOSRC!!-,No%20plurals,-In%20go%2C%20package. 

Бувають кейси коли варто використовувати назву в множині. Наприклад в контексті стандартної бібліотеки `bytes.Equal` набагато кращий вибір ніж `byte.Equal` оскільки цей пакет оперує набором байт.